### PR TITLE
Update automation calculator

### PR DESCRIPTION
**Pull request by jamestalton**
https://github.com/ansible/ansible-ui/assets/6277895/e36228ea-79bb-4bd3-afa9-330a45a5907f

### CODE REVIEW

**Comment by skochay330** [comment]
A couple of thoughts: 
- Do we want to keep the tags on the right of the page header? Looks a little odd to me, but maybe UX can guide us @trahman73 @tiyiprh @aratti96 
- Default sort option should be 'Savings from successful hosts' (`successful_hosts_savings`)
- Can we make the dropdown lists scrollable? Currently if there are many items, the list goes on and overflows past the footer (see below)

https://github.com/ansible/ansible-ui/assets/89094075/fbda3266-41ff-4155-a437-1dc13742063a

**Comment by tiyiprh** [comment]
> A couple of thoughts:
> 
> * Do we want to keep the tags on the right of the page header? Looks a little odd to me, but maybe UX can guide us @trahman73 @tiyiprh @aratti96

It was brought up that the tags being on a separate line was taking up space for the rest of the page and charts so our suggestion was to move it to the top right in the header. If this doesn't look right to others, we're happy to move this back to its original location.

> * Can we make the dropdown lists scrollable? Currently if there are many items, the list goes on and overflows past the footer (see below)

There needs to be a maximum height for the dropdown and the rest should be scrollable. Should our lookup functionality apply here vs the dropdown?

**Comment by jamestalton** [comment]
> Can we make the dropdown lists scrollable? Currently if there are many items, the list goes on and overflows past the footer (see below)

Yes. Will do.

**Comment by jamestalton** [comment]
The fix for the dropdown lists being scrollable will be taken care of as part of another PR. We are moving away from the deprecated PF4 Select component to the new PF4/next  Select component. We have a wrapper for that called PageSingleSelect and PageMultiSelect. Those new base components support the scrollable dropdown and the toolbar filters will be updated to use those components in a future PR.

PR: For the PageMultiSelect: https://github.com/ansible/ansible-ui/pull/753

File: frontend/awx/analytics/Reports/Reports.tsx [hunk 1]
```diff
@@ -76,7 +56,7 @@ export default function Reports() {
   }
 
   const reportTags = (
-    <LabelGroup numLabels={6}>
+    <LabelGroup numLabels={6} style={{ flexWrap: 'nowrap' }}>
```
Comment (ZitaNemeckova) [inline frontend/awx/analytics/Reports/Reports.tsx:59]: Use styledComponent.

File: frontend/awx/analytics/Reports/ErrorStates.tsx [hunk 2]
```diff
@@ -4,7 +4,6 @@ import { ReactElement } from 'react';
 import { useTranslation } from 'react-i18next';
 import { EmptyStateCustom } from '../../../../framework/components/EmptyStateCustom';
 import { useActiveUser } from '../../../common/useActiveUser';
-import Picture from '../../assets/images/ansible_analytics_screenshot.png';
```
[the remark this answers is not in the document]
Comment (jamestalton) [reply frontend/awx/analytics/Reports/ErrorStates.tsx:7]: We should not include screenshots in the UI.

- It makes the UI download size bigger than needed.
- Over time those screen shots get outdated.
- The screenshots do not match the users color scheme.

If we need to talk to the PM about this let me know.

File: frontend/awx/analytics/Reports/Reports.tsx (hunk 1 above)
Comment (jamestalton) [reply frontend/awx/analytics/Reports/Reports.tsx]: Done

File: frontend/awx/analytics/Reports/ErrorStates.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ZitaNemeckova) [reply frontend/awx/analytics/Reports/ErrorStates.tsx:7]: Let's check with PM to make sure.

[the remark this answers is not in the document]
Comment (ZitaNemeckova) [reply frontend/awx/analytics/Reports/ErrorStates.tsx:7]: Resolved.

**Review by ZitaNemeckova — COMMENTED** [review]
Tested in UI. Found just one minor issue. 

Set `per page` to have multiple pages ->  Go to last page -> Select a filter that will return no items or less items than `per page` -> See pagination

Screenshot 2023-08-07 at 16 47 33

Otherwise LGTM :+1:

File: frontend/awx/analytics/Reports/ErrorStates.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamestalton) [reply frontend/awx/analytics/Reports/ErrorStates.tsx:7]: Talked with PM. We will be updating the empty analytics state to be a component with more details about analytics and how to get starts + links.

**Comment by jamestalton** [comment]
Added check for page out of range.

**Review by ZitaNemeckova — APPROVED** [review]

**Pull request merged**